examples/memoize-simple.yaml

```diff
@@ -37,7 +37,7 @@ spec:
     container:
       image: docker/whalesay:latest
       command: [sh, -c]
-      args: ["cowsay {{inputs.parameters.message}} > /tmp/hello_world.txt"]
+      args: ["cowsay world > /tmp/hello_world.txt"]
     outputs:
```

Comment:
We don't need to parameterize this anymore? (just trying to follow along since this memoization capability is new to me)

Reply:
Here we only want to parameterize the key in the cache so that we can clearly see that the `memoize.key` is the key in the configmap.

Reply:
So, basically does the template not get run in this case since the output parameter is named "hello" and the "hello" key is memoized?

Unfortunately, I had a hard time comprehending this - I wonder if we should add more [details](https://argoproj.github.io/argo-workflows/memoization/) to the page on memoization?

Reply:
Yes it will be skipped if the cache is already created and not outdated (based on `maxAge`).

Reply:
got it, thanks!